cmo-module/src/main/resources/xsl/mei/incip.xsl

```diff
@@ -49,7 +49,7 @@
         <xsl:with-param name="content2">
           <xsl:value-of select="." disable-output-escaping="yes"/>
         </xsl:with-param>
-        <xsl:with-param name="content2Arab" select="contains(@xml:lang,'-arab')" />
+        <xsl:with-param name="content2Arab" select="contains(@xml:lang,'-arab') or contains(@xml:lang, '-latn')" />
```

Comment:
"rtl" should only be used for arab. Using this html attribut for latin is wrong.